doc/api/http.markdown

```diff
@@ -64,10 +64,10 @@ a `'close'` event or a special `'agentRemove'` event. This means that if
 you intend to keep one HTTP request open for a long time and don't
 want it to stay in the pool you can do something along the lines of:
 
-    http.get(options, function(res) {
+    http.get(options, (res)=> {
```

Comment:
It's definitely a brand new :bike::house_with_garden:; I don't feel strongly enough either way to propose setting a hard-and-fast rule for it. In my own projects, I lean towards minimizing characters in code examples to give other info more room, but two characters is not a _huge_ amount of savings so I'm comfortable leaving this as a personal preference of the doc author for the time being.